WooCommerce/Classes/POS/Dependencies/POSAnalyticsAdapter.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import WooFoundation
+
+/// Adapter that implements POSAnalyticsProviding using ServiceLocator
+final class POSAnalyticsAdapter: POSAnalyticsProviding {
+    func track(event: WooAnalyticsEvent) {
+        let mainAppEvent = WooAnalyticsEvent(statName: event.statName, properties: event.properties, error: event.error)
+        ServiceLocator.analytics.track(event: mainAppEvent)
```

Comment:
maybe I'm missing something, can we just track `event: WooAnalyticsEvent` directly?
```suggestion
        ServiceLocator.analytics.track(event: event)
```

Reply:
I think we can. Good observation. It's a leftover from the implementation when I had two `WooAnalyticsEvent` implementations.